Description: Make QtWebEngine optional
 Most resources do not depend on QtWebEngine, so let's enable kdepim-runtime for
 all other archs, too.
Author: hefee <hefee@debian.org>
Origin: Debian
Last-Update: 2019-03-21
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt	2021-12-22 19:05:59.901592857 +0000
+++ b/CMakeLists.txt	2021-12-22 19:09:30.140987017 +0000
@@ -105,8 +105,8 @@
 if(Qt5TextToSpeech_FOUND)
 set(HAVE_TEXT_TO_SPEECH_SUPPORT TRUE)
 endif()
-find_package(Qt5 ${QT_REQUIRED_VERSION} CONFIG REQUIRED Network Widgets Test XmlPatterns DBus WebEngineWidgets NetworkAuth)
-
+find_package(Qt5 ${QT_REQUIRED_VERSION} CONFIG REQUIRED Network Widgets Test XmlPatterns DBus NetworkAuth)
+find_package(Qt5 OPTIONAL_COMPONENTS WebEngineWidgets)
 
 # KF5 package
 find_package(KF5Config ${KF5_MIN_VERSION} CONFIG REQUIRED)
--- a/resources/CMakeLists.txt
+++ b/resources/CMakeLists.txt
@@ -64,5 +64,8 @@ add_subdirectory( vcarddir )
 add_subdirectory( icaldir )
 add_subdirectory( vcard )
 add_subdirectory( folderarchivesettings )
-add_subdirectory( tomboynotes )
-add_subdirectory( ews )
+
+if(Qt5WebEngineWidgets_FOUND)
+  add_subdirectory( ews )
+  add_subdirectory( tomboynotes )
+endif()
